action.yml

```diff
@@ -2,73 +2,73 @@ name: 'Datadog Synthetics CI'
 description: 'Run Datadog Synthetics CI tests as part of your Github Actions workflow'
 author: 'Datadog'
 inputs:
-  api_key:
+  api-key:
     required: true
     description: 'Datadog API key.'
-  app_key:
+  app-key:
     required: true
     description: 'Datadog Application key.'
-  batch_timeout:
+  batch-timeout:
     required: false
     description: 'The duration (in milliseconds) after which the batch fails as timed out.'
-  config_path:
+  config-path:
     required: false
     description: 'Path to global configuration JSON.'
-  datadog_site:
+  datadog-site:
     required: false
     description: 'Datadog site.'
-  fail_on_critical_errors:
+  fail-on-critical-errors:
     required: false
     description: 'Fail the CI job if no tests were triggered, or results could not be fetched from Datadog.'
-  fail_on_missing_tests:
+  fail-on-missing-tests:
     required: false
     description: 'Fail the CI job if at least one specified test with a public ID (using `public_ids` or listed in a test file) is missing in a run (for example, if it has been deleted programmatically or on the Datadog site).'
-  fail_on_timeout:
+  fail-on-timeout:
     required: false
     description: 'Fail the CI job if at least one test exceeds the default test timeout.'
   files:
     required: false
     description: 'Glob pattern to detect Synthetic tests config.'
-  junit_report:
+  junit-report:
     required: false
     description: 'The filename for a JUnit report if you want to generate one.'
-  public_ids:
+  public-ids:
     required: false
     description: 'Public IDs of Synthetic tests to run, separated by new lines or commas. If no value is provided, tests are discovered in `*.synthetics.json` files.'
   subdomain:
     required: false
     description: 'Name of your Datadog custom subdomain.'
-  test_search_query:
+  test-search-query:
     required: false
     description: 'Search query to trigger tests.'
   tunnel:
     required: false
     description: 'Enable tunnel to interact with Datadog API.'
-    default: false
+    default: 'false'
```

Comment:
The spec expects values to be strings. Then, `core.getBooleanInput()` converts the string to a bool, based on the [YAML spec](https://yaml.org/spec/1.2-old/spec.html#id2804923)

![image](https://github.com/user-attachments/assets/d41aefe3-907c-44c1-ae0a-93a65e335020)